<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>710</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>634</width>
    <height>834</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SHA1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>591</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>SHA-1</string>
    </property>
    <widget class="QPushButton" name="SHA_oblicz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Oblicz sumę</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="SHA_wynik">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="SHA_porownaj2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Wklej lub wczytaj sumę kontrolną do porównania</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="SHA_wynik2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="placeholderText">
      <string>Wynik porównania</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SHA_porownaj">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Porównaj z:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MD5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>591</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>MD5</string>
    </property>
    <widget class="QPushButton" name="MD5_oblicz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Oblicz sumę</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="MD5_wynik">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="MD5_porownaj_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Wklej lub wczytaj sumę kontrolną do porównania</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="MD5_wynik2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="placeholderText">
      <string>Wynik porównania</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MD5_porownaj">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Porównaj z:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CRC32">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>591</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>CRC-32</string>
    </property>
    <widget class="QPushButton" name="CRC32_oblicz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Oblicz sumę</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="CRC32_wynik">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="CRC32_porownaj_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Wklej lub wczytaj sumę kontrolną do porównania</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="CRC32_wynik2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>591</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="placeholderText">
      <string>Wynik porównania</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CRC32_porownaj">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Porównaj z:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOtw_rz">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtw_rz"/>
    <addaction name="actionZapisz_sumy_kontrolne"/>
    <addaction name="actionWczytaj_sumy_kontrolne"/>
    <addaction name="actionZamknij"/>
   </widget>
   <addaction name="menuOtw_rz"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOtw_rz">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionZapisz_sumy_kontrolne">
   <property name="text">
    <string>Zapisz sumy kontrolne..</string>
   </property>
  </action>
  <action name="actionWczytaj_sumy_kontrolne">
   <property name="text">
    <string>Wczytaj sumy kontrolne..</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
